doc/operators/start_network.rst

```diff
@@ -9,41 +9,65 @@ Starting a New Network
 Starting the First Node
 -----------------------
 
-To create a new CCF network, the first node of the network should be started with the ``start`` option:
+To create a new CCF network, the first node of the network should be invoked with the ``start`` option:
 
 .. code-block:: bash
 
     $ cchost
     --enclave-file /path/to/enclave_library
-    --node-address node_ip:node_port
-    --rpc-address <ccf-node-address>
+    --node-address <ccf-node-address>
+    --rpc-address <ccf-node-rpc-address>
```

Comment:
The RPC address "<ccf-node-address>" is referred to everywhere else in our docs. I think it will need updating too.